bazarr/languages/get_languages.py

```diff
@@ -57,10 +57,11 @@ def create_languages_dict():
 def audio_language_from_alpha2(lang):
```

Comment:
This function name probably don't make sense anymore since lang is not an alpha2. Let's rename it,